Doc/library/pickle.rst

```diff
@@ -30,9 +30,19 @@ avoid confusion, the terms used here are "pickling" and "unpickling".
 
 .. warning::
 
-   The :mod:`pickle` module is not secure against erroneous or maliciously
-   constructed data.  Never unpickle data received from an untrusted or
-   unauthenticated source.
+   The ``pickle`` module **is not secure**. Only unpickle data you trust.
+
+   It is possible to construct malicious pickle data which will **execute
+   arbitrary code during unpickling**. Never unpickle data that could have come
```

Comment:
Maybe:
```suggestion
   arbitrary code during pickle.load[s]**. Never unpickle data that could have come
```

Reply:
I prefer the "unpickling version"

Reply:
To me, using "pickle.load[s]" here creates the false impression that it's just the module level convenience functions which are insecure. It incorrectly implies (by omission) that "p = pickle.Unpickler(..); p.load()" would be safe.

Reply:
I agree. "unpickling" is defined immediately before this warning, in the previous paragraph.

Reply:
My initial thinking was about making it _more_ explicit but since unpickling is already defined and `pickle.load[s]` doesn't cover all the cases it totally makes sense to reject this suggestion :).

Thanks for the great points!